az-report-ready/systemd/az-report-ready.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Report VM as ready
+After=network.target
```

Comment:
I think we should use

```
After=network-online.target
Wants=network-online.target
```

Like explained here: https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/

Otherwise network interfaces might be unconfigured at this point.

Hopefully it works on AZ VMs :)